Workaround a problem in readline's autoconf system. readline incorrectly
attempts to compile and run a test where simply compiling is quite sufficient.
Since you can't run tests when cross compiling this is a bit of a bugger. This
patch pre-caches the result so as to avoid the broken test from being run. This
problem is apparantly fixed upstream and should go away when gdb is upgraded to
readline 5.0.


Signed-off-by: Chris Smith <chris.smith@st.com>

--- gdb-6.3-orig/objdir/readline/config.cache	1970-01-01 01:00:00.000000000 +0100
+++ gdb-6.3/objdir/readline/config.cache	2004-12-07 10:17:04.763001000 +0000
@@ -0,0 +1 @@
+bash_cv_have_mbstate_t=${bash_cv_have_mbstate_t=yes}
